crypto/x509v3/v3_admis.c

```diff
@@ -202,3 +201,113 @@ static int i2r_ADMISSION_SYNTAX(const struct v3_ext_method *method, void *in,
 err:
     return -1;
 }
+ 
+const GENERAL_NAME* ADMISSION_SYNTAX_get0_admissionAuthority(const ADMISSION_SYNTAX *as)
+{
+    return as->admissionAuthority;
+}
+ 
+void ADMISSION_SYNTAX_set0_admissionAuthority(ADMISSION_SYNTAX *as, GENERAL_NAME *aa)
+{
+    GENERAL_NAME_free(as->admissionAuthority);
+    as->admissionAuthority = aa;
+}
+ 
+const STACK_OF(ADMISSIONS)* ADMISSION_SYNTAX_get0_contentsOfAdmissions(const ADMISSION_SYNTAX *as)
+{
+    return as->contentsOfAdmissions;
+}
+ 
+void ADMISSION_SYNTAX_set0_contentsOfAdmissions(ADMISSION_SYNTAX *as, STACK_OF(ADMISSIONS) *a)
+{
+    sk_ADMISSIONS_free(as->contentsOfAdmissions);
```

Comment:
Need sk_ADMISSIONS_pop_free() here or there will be leaks.

Reply:
done